stagecraft/apps/dashboards/tests/views/test_module.py

```diff
@@ -140,13 +127,6 @@ def test_get_module_by_uuid_404s_when_user_not_owner_of_dashboard(self):
 
         assert_that(resp.status_code, is_(equal_to(404)))
 
-    def test_modules_on_dashboard_doesnt_delete(self):
```

Comment:
I think it would be good to have a test to see if this route just deletes all the modules on the dashboard, whether it does nothing, or whether it delete the dashboard. I'm not confident I can say which one it is - I find the resource view too confusing.